### DISCUSSION TITLE
buddhism/linji/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#8] [General]
# buddhism/linji/

https://shanenull.com/buddhism/linji/

**Comment by shane0** [comment]
https://youtu.be/YVv3vqLqUS4?si=T4FhrSAQkH0hDmhO

**Comment by shane0** [comment]
Bandied means to be used or discussed often in a casual or careless way. It can also mean to be thrown back and forth between people.

In the Record of Linji, the phrase "bandied words" refers to the empty and meaningless words that are often used in Buddhist discourse. Linji was a Zen master who was known for his direct and blunt teaching style. He often criticized the use of bandied words, saying that they were a hindrance to true spiritual progress.

For example, in one passage in the Record of Linji, he says:

> "When you study the teachings of the Buddhas and Patriarchs, you should not be attached to bandied words. You should not say, 'This is right' or 'This is wrong.' You should not say, 'This is good' or 'This is bad.' You should simply let go of all thoughts and concepts and see things as they really are."

Linji's point is that the true meaning of the Buddhist teachings cannot be conveyed through words. It can only be experienced directly through meditation and practice. When we become attached to bandied words, we get caught up in intellectual concepts and miss the essential point.

Another way to understand the phrase "bandied words" is to think of it as referring to the clichés and platitudes that are often used in religious discourse. These words are often used without much thought or reflection, and they can easily become meaningless. Linji is warning us to be careful not to get caught up in this kind of empty talk.

Instead, we should focus on our own personal experience and understanding of the Buddhist teachings. We should not be afraid to question everything and to think for ourselves. Only then can we truly hope to gain insight into the true nature of reality.

**Comment by shane0** [comment]
https://youtu.be/B17xj_lyBnk?si=HjJ9suOmyV7F7ElA
